app/views/donation_sites/index.html.erb

```diff
@@ -93,7 +93,7 @@
   <%= render(
           layout: "shared/csv_import_modal",
           locals: {
-              title: 'Import Donation Sites',
+              title: 'Donation Sites',
```

Comment:
I like the solution a lot better - I'd just suggest changing the name of this variable from "Title" to something like "import\_type" or something similar, since it no longer represents the title.

Reply:
Addressed in [6feae6d](https://github.com/rubyforgood/human-essentials/pull/3492/commits/6feae6d78825bf5bef1384c080b2a5a257634833)